src/core/raster/qgsrasterlayer.h

```diff
@@ -405,9 +405,10 @@ class CORE_EXPORT QgsRasterLayer : public QgsMapLayer, public QgsAbstractProfile
 
     /**
      * \brief Refresh renderer with new extent, if needed
-     *  \note not available in Python bindings
+     * \returns true if the renderer has been refreshed.
```

Comment:
```suggestion
     * \returns TRUE if the renderer has been refreshed.
```

(ensures correct formatting of the literal in the api docs)

Reply:
Fixed.